tests/unit/GithubUtilsTest.ts

```diff
@@ -1,15 +1,72 @@
 /**
  * @jest-environment node
  */
-const core = require('@actions/core');
-const GithubUtils = require('../../.github/libs/GithubUtils');
+import * as core from '@actions/core';
+import GithubUtils from '../../.github/libs/GithubUtils';
 
 const mockGetInput = jest.fn();
 const mockListIssues = jest.fn();
 
+type DeployBlockers = {
+    url: string;
+    number: number;
+    isResolved: boolean;
+};
+
+type ExpectedReponse = {
+    PRList: Array<{
+        url: string;
+        number: number;
+        isVerified: boolean;
+    }>;
+    labels: Array<{
+        color: string;
+        default: boolean;
+        description: string;
+        id: number;
+        name: string;
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        node_id: string;
+        url: string;
+    }>;
+    tag: string;
+    title: string;
+    url: string;
+    number: number;
+    deployBlockers: DeployBlockers[];
+    internalQAPRList: string[];
+    isTimingDashboardChecked: boolean;
+    isFirebaseChecked: boolean;
+    isGHStatusChecked: boolean;
+};
+
+type Issue = {
+    url: string;
+    title: string;
+    labels: Array<{
+        id: number;
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        node_id: string;
+        url: string;
+        name: string;
+        color: string;
+        default: boolean;
+        description: string;
+    }>;
+    body: string;
+};
+
+type ObjectMethodData<T> = {
+    data: T;
+};
+
+type Mutable<T> = {-readonly [P in keyof T]: T[P]};
+
+const asMutable = <T>(value: T): Mutable<T> => value as Mutable<T>;
+
 beforeAll(() => {
     // Mock core module
-    core.getInput = mockGetInput;
+    asMutable(core).getInput = mockGetInput;
 
     // Mock octokit module
     const moctokit = {
```

Comment:
Is possible to use a type from GitHub library here?

Reply:
I couldn't find a way to make the mock complaint with the GitHub lib typing.
There was too a lot of missing fields which we do not use on the unit test that needed to be populated and couldn't find a solution for all of them, the best way I think is to have GithubUtils correctly typed so this one would automatically get the correct types.